Core/src/IfcGeometryConverter/SolidModelConverter.h

```diff
@@ -2235,7 +2235,7 @@ namespace OpenInfraPlatform
 				carve::geom::vector<3> baseSurfacePosition;
 				carve::math::Matrix basePositionMatrix(carve::math::Matrix::IDENT());
 				
-				placementConverter->getPlane(elemBaseSurface->Position.lock(), baseSurfacePlane, baseSurfacePosition, this->UnitConvert()->getLengthInMeterFactor());
+				placementConverter->getPlane(elemBaseSurface->Position.lock(), baseSurfacePlane, baseSurfacePosition);

```

Comment:
You can remove the `.lock()` since we're dealing with `EXPRESSReference` now. Apply appropriately for the whole PR.

Reply:
Although, I'm unsure if that is the syntax for optionals. Let's have a call if it is not obvious.

Reply:
Everything is working fine without "lock()"